Sources/ArcGISToolkit/Common/AttachmentList.swift

```diff
@@ -95,5 +100,8 @@ struct AttachmentLoadButton: View  {
             .frame(width: 24, height: 24)
             .padding(.leading)
         }
+        .alert(isPresented: $downloadAlertIsPresented) {
+            Alert(title: Text.emptyAttachmentDownloadErrorMessage)
+        }
```

Comment:
FYI: This API is deprecated.

Reply:
Ah, thank you. Not sure how I missed that. #794

Reply:
You're welcome.